src/ngraph/runtime/executable.hpp

```diff
@@ -73,14 +73,6 @@ class NGRAPH_API ngraph::runtime::Executable
     /// \returns  preferred pipeline_depth
     virtual size_t get_preferred_pipeline_depth() const;
 
-    /// \brief Set the input Parameters
-    /// \param params ngraph::ParameterVector of all input parameters
-    void set_parameters(const ngraph::ParameterVector& params);
```

Comment:
these APIs were originally added by @jiminha in r0.25 for a specific use-case in nnp-i transformer, can we work with her to close on that use-case before reverting.